cypress/e2e/teams/teacherAddsEditsDeleteTeamEventWithVideoConference.feature

```diff
@@ -1,11 +1,17 @@
-@unstable_test
+@stable_test
+@release
 Feature: Team - Teacher adds edits and deletes team event with video conference and student can participate as an internal team member
 
   As a teacher I want to add video conference to the team event so that team members can collaborate via video conference.
 
-  @unstable_test
-  Scenario: as a pre condition Admin enables the video conference option for a team event
-    Given I am logged in as a 'admin1_dbc' at 'dbc'
+  @stable_test
```

Comment:
Why there is `@stable_test` tag when we already mentioned it in the higher level?